docs/reference/commandline/ps.md

```diff
@@ -50,6 +50,8 @@ The currently supported filters are:
 * name (container's name)
 * exited (int - the code of exited containers. Only useful with `--all`)
 * status (created|restarting|running|paused|exited)
+* ancestor (image id) - filters containers that were created from the
```

Comment:
@thaJeztah oh right, we should probably just add `image` (so it stays general) there and add more example in a `Filtering` chapter (as discussed a bit on IRC :wink:)